packages/@okta/vuepress-site/docs/guides/social-login/main/microsoft/micronote.md

```diff
@@ -0,0 +1 @@
+> **Note:** When you configure a Microsoft social IdP, Microsoft accounts from *any tenant* can sign in using this IdP. A Microsoft social IdP uses Microsoft's common OpenID Connect endpoints on the Microsoft identity platform. Because of this, tenant-level isolation isn't possible. To configure an Azure/Entra IdP and/or to limit user sign-in flows to a specific Microsoft tenant, use the [enterprise IdP instructions](/docs/guides/add-an-external-idp/openidconnect/main/) instead.
```

Comment:
```suggestion
> **Note:** When you configure a Microsoft social IdP, Microsoft accounts from *any tenant* can sign in using this IdP. A Microsoft social IdP uses Microsoft's common OpenID Connect endpoints on the Microsoft identity platform. Because of this, tenant-level isolation isn't possible. To configure an Azure/Entra IdP and/or to limit user sign-in flows to a specific Microsoft tenant, use the [enterprise IdP instructions](/docs/guides/add-an-external-idp/azure/main/) instead.
```